validations/src/tests.rs

```diff
@@ -2857,11 +2863,16 @@ fn dr_pool_with_dr_in_tally_stage_no_commits() -> (DataRequestPool, Hash, Public
         DataRequestStage::TALLY
     );
 
-    (dr_pool, dr_pointer, dr_pkh)
+    (dr_pool, dr_pointer, dr_output, dr_pkh)
 }
 
-fn dr_pool_with_dr_in_tally_stage_no_reveals(
-) -> (DataRequestPool, Hash, PublicKeyHash, Vec<PublicKeyHash>) {
+fn dr_pool_with_dr_in_tally_stage_no_reveals() -> (
```

Comment:
I think all these tests could use a human friendly description of the setup they are reproducing.

If you think it's also a good idea, I can make suggestions for them (although I will have to discuss with you, as some of them are not evident to me).

Reply:
Should we do in other issue? To do not delay the patch for the testnet

Reply:
My concern is that I can't verify the completeness of the tests without fully understanding the setups being tested.

Reply:
Awesome comments. Marking as resolved.